inputs/flexibility/import_export/co2_emissions_of_imported_electricity_future.ad

```diff
@@ -1,8 +1,17 @@
-#The user insert g/KWH and the ETM processes kg/MJ. For this reason we divide it by 1000 and with MJ_PER_KWH
+# The co2_conversion_per_mj defined in the FCE-module can only be updated by
+# updating shares. Therefore, we have to calculate the shares of renewable electricity 
+# (0 g/kWh) and fossil electricity (1000 g/kWh) and then update the FCE.
 
-- query = UPDATE(CARRIER(imported_electricity), co2_conversion_per_mj, USER_INPUT() / 1000 / MJ_PER_KWH )
+- query =
+    share_fossil_electricity = USER_INPUT() / 1000;
```

Comment:
Dividing through integers is dangerous! Please use 1000.0 or, even better, `THOUSANDS`

Reply:
Yes! I will use 1000.0 though, because this is the max slider value. I think it is going to be a bit intransparent when we would use `THOUSANDS`. When people want to update the slider_max its maybe not clear that this value also has to be updated